tests/extraction_tests/test_round_three_extraction.py

```diff
@@ -383,17 +383,6 @@ def test_extract_outcomes_with_invalid_project(mock_outcomes_sheet, mock_project
     )
 
 
-def test_extract_outcomes_with_null_project(mock_outcomes_sheet, mock_project_lookup, mock_programme_lookup):
```

Comment:
This test needs to come back for round 4 it is deleted because we are preserving old functionality

Reply:
Can the above change be implemented for Round 4 and then this test be moved to the Round 4 test module